flutter-studio/src/io/flutter/profiler/SnapshotData.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.profiler;
+
+import com.android.tools.adtui.model.DefaultDurationData;
+
+public final class SnapshotData extends DefaultDurationData {
+  public SnapshotData(long duration) {
+    super(duration);
+  }
+
+  @Override
+  public String toString() {
+    return "Reset Event";
```

Comment:
This toString seems like a bug. I saw another toString that was reset event. Perhaps remove these toStrings unless they impact the UI.

Reply:
Diito on all these toStrings removed.